datadog/resource_datadog_synthetics_test_test.go

```diff
@@ -119,7 +119,7 @@ func TestAccDatadogSyntheticsBrowserTest_importBasic(t *testing.T) {
 		CheckDestroy: testSyntheticsTestIsDestroyed(accProvider),
 		Steps: []resource.TestStep{
 			{
-				Config: createSyntheticsBrowserTestConfig(testName),
+				Config: createSyntheticsBrowserTestConfig(testName, false),
```

Comment:
No steps for import since some steps require some info from the backend

Reply:
I think in this case we can test it, so we should.